core/new-gui/src/app/workspace/service/operator-metadata/mock-operator-metadata.data.ts

```diff
@@ -1,96 +1,101 @@
+// import functions from lodash: import individual functions from lodash-es
+//  to avoid include the entire lodash library
+import cloneDeep from 'lodash-es/cloneDeep';
 import { OperatorSchema, OperatorMetadata, GroupInfo } from '../../types/operator-schema';
 
 /**
- * MOCK_OPERATOR_METADATA const variable is a mock operator metadata consisting of
- *  a few operators, which can be useful for testing.
+ * Exports constants related to operator schema and operator metadata for testing purposes.
  *
  */
 
-export const MOCK_OPERATOR_SCHEMA_LIST: OperatorSchema[] = [
+export const getMockOperatorSchemaList: () => OperatorSchema[] =
+  () => cloneDeep([
```

Comment:
Why the change to cloneDeep?

Reply:
To avoid a test case changing the global constant variable and affect the all other test cases. In this way, test cases only get a copy of the data. It won't have any effect even if they modified the data.

Reply:
I have added the reason to the PR description.